docs/build/javascript/quick-start.md

```diff
@@ -0,0 +1,144 @@
+# Quickstart
+Learn the basics by setting up and interacting with the [Ceramic CLI](./cli/installation.md)
+
+!!! warning ""
+    **Want an even faster way to try Ceramic? Visit the [Playground Demo App](https://playground.ceramic.dev){:target="_blank"} to test the full stack of Ceramic Components in the browser.
+
+## **1. Install the Client**
+
+Visit the [Ceramic Client](./javascript/installation.md) page for instructions on how to quickly install the Client.
+
+## **2. Create a stream**
+Creating streams is dependent on the StreamType of the record. More details can be found in our [StreamTypes Overview](../streamtypes/overview.md). A basic sample can be found below.
+
+=== "Command"
+
+    ```JavaScript
+    import { TileDocument } from '@ceramicnetwork/stream-tile'
+
+    const schemaId = 'k3y52l7qbv1fry1fp4s0nwdarh0vahusarpposgevy0pemiykymd2ord6swtharcw' // this is our Basic Profile Schema.
+    
+    const content = {
+      foo: 'bar'
+    }
+    const metadata = {
+      controllers: [ceramic.did.id],
+      family: 'Document Family',
+      schema: schemaId
+    }
+
+    const doc = await TileDocument.create(ceramic, content, metadata)
+
+    const streamId = doc.id.toString()
+    ```
+
+## **3. Query a stream**
+Use the `loadStream()` function to query the current [state](../learn/glossary.md#state) of a stream. You will need to provide it's *StreamID*
+
+=== "Command"
+
+    ```JavaScript
+    const streamId = "kjzl6cwe1jw147ww5d8pswh1hjh686mut8v1br10dar8l9a3n1wf8z38l0bg8qa"
+    const stream = await ceramic.loadStream(streamId)
```

Comment:
```suggestion
    const stream = await TileDocument.load(streamId)
```

Reply:
Using the StreamType-specific load method will let you use StreamType specific functionality.  For example, the way streams get updated is streamtype-specific, so if you want to be able to access the TileDocument's `.update()` method to update the stream after loading it, you need to specify the type.  The default `Stream` type that's returned by `loadStream()` doesn't include any update functionality (or any other streamtype-specific accessors)